README.md

```diff
@@ -8,3 +8,157 @@
+
+```
+github "velocidi/velocidi-ios-objc-sdk" ~> 0.0.1
+```
+
+Then, run `carthage` to build the framework and drag the built VelocidiSDK.framework into your Xcode project.
+
+## Requirements
+
+VelocidiSDK should work with any version of iOS equal or bigger than 11.0.
+
+## Usage
+
+### Setup
+
+Initialize the VelocidiSDK with the necessary `trackingHost` and the `matchHost` URLs. Without this, VelocidiSDK will not work. We suggest doing this when the application launches.
+
+__Swift__
+```swift
+import VelocidiSDK
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+
+        let config = VSDKConfig(hosts: "https://tr.yourdomain.com", "https://match.yourdomain.com")!
+        VSDKVelocidi.start(config)
+        return true
+    }
+```
+
+__Objective-C__
+```objc
+@import VelocidiSDK;
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // Override point for customization after application launch.
+    
+    VSDKConfig * config = [[VSDKConfig alloc] initWithHosts:@"https://tr.yourdomain.com": @"https://match.yourdomain.com"];
+    [VSDKVelocidi start: config];
+    return YES;
+}
+```
+
+### Send a track event
+
+In order to sent a tracking event, create an instance of TrackingEvent. Then, call the singleton instance of `VSDKVelocidi` and use the method `track`
+
+__Swift__
+```swift
+import VelocidiSDK
+
+...
+
+let trackingEvent = VSDKPageView()
+trackingEvent.siteId = "RandomSiteId"
+trackingEvent.clientId = "RandomClientId"
+
+VSDKVelocidi.sharedInstance().track(trackingEvent)
+```
+
+__Objective-C__
+```objc
+@import VelocidiSDK;
+
+...
+
+VSDKTrackingEvent * trackingEvent =  [[VSDKPageView alloc] init];
+trackingEvent.clientId = @"RandomSiteId";
+trackingEvent.siteId = @"RandomClientId";
+
+[VSDKVelocidi.sharedInstance track: trackingEvent] 
+```
+
+
+You can also pass callback blocks that will that will be called when the request is successful or if if fails.
+
+__Swift__
+```swift
+VSDKVelocidi.sharedInstance().track(trackingEvent, onSuccess:{ (response: URLResponse, responseObject: Any) in
+  print("Success! Response: \(response)")
+}, onFailure:{(error: Error) in
+  print("Failed! Error: \(error.localizedDescription)")
+})
+```
+
+__Objective-C__
+```objc
+[VSDKVelocidi.sharedInstance track: trackingEvent onSuccess: ^(NSURLResponse * response, id responseObject){
+    NSLog(@"Success! Response: %@", trackingNumber);
+} onFailure: ^(NSError * error){
+    NSLog(@"Failed! Error: %@", [error localizedDescription]);
+}];
+```
+
+#### Available tracking events model classes
```

Comment:
Don't really like this. I think it's strange to have a link in a README header. If I did not know I would assume it would be a link to copy the link, similar to this 
![image](https://user-images.githubusercontent.com/7612420/58255096-1c821700-7d64-11e9-9bf4-e392336d3905.png)

Reply:
Hum... I would not have that impression. Anyway, with the list below I don't find it necessary to have a link there. It would be a "nice to have".